libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -146,8 +146,6 @@ string ABIFunctions::tupleEncoderPacked(
 		functionName += "_reversed";
 
 	return createFunction(functionName, [&]() {
-		solAssert(!_givenTypes.empty(), "");
```

Comment:
Which test case is this fixing? I can't see an empty `encodePacked()` in any of them.  #10390 refers to `SolidityEndToEndTest/abi_encodePacked` which does not seem to be touched in this PR? Did it mean `SolidityEndToEndTest/abi_encodePacked*` ?

Reply:
Thanks, it seems I forgot to enable SolidityEndToEndTest/abi_encodePacked